### PR TITLE
Bugfix for webpack-dev-server 3.1.11

### PR DESCRIPTION
**Pull request by jlieb10**
Upgrading webpack-dev-server to 3.1.11 causes constant console logging due to new security features. The current best supported work around is to add the `--disable-host-check` flag to the 
script in `package.json`

Please see webpack-dev-server's documentation of this issue [**here**](https://github.com/webpack/webpack-dev-server/issues/1604)

**TBD**: check to see if this flag is still necessary next time we upgrade webpack-dev-server

### CODE REVIEW

File: support-frontend/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "build-ssr": "webpack --config webpack.ssr.js && node scripts/ssrScript.js",
     "build-prod": "npm-run-all clean validate test storybook-static 'webpack --config webpack.prod.js --env.prod'",
     "webpack": "webpack",
-    "webpack-dev-server": "webpack-dev-server --config webpack.dev.js --port 9211 --public support.thegulocal.com",
+    "webpack-dev-server": "webpack-dev-server --config webpack.dev.js --port 9211 --public support.thegulocal.com --disable-host-check",
```
Comment (johnduffell) [inline support-frontend/package.json]: is it possible to add a comment with a link to the issue jonny was looking at?

**Comment by johnduffell** [comment]
good, but see my comment. although it can be linked back to the PR, it might be nice to comment inline so it is more obvious.
Also, Rupert is looking at the master build failure.
👍 other than that thanks!

File: support-frontend/package.json (hunk 1 above)
Comment (jlieb10) [reply support-frontend/package.json]: @johnduffell comments are not supported in json files so we settled for a PR with the documentation 😢

**Comment by jlieb10** [comment]
@johnduffell your PR was definitely not the source of the issue - you just brought the `yarn.lock` file up to date with the packages that we are currently using, which should've happened when the last package was updated

Also, unfortunately inline comments are not supported in `json` files

**Comment by jlieb10** [comment]
Related PR: https://github.com/guardian/support-frontend/pull/1774

**Pull request merged**